docs/LocalDev.md

```diff
@@ -26,11 +26,11 @@ We recommend using Visual Studio Code (cross platform) or Visual Studio 2019 (Wi
 You can of course use VIM if you are hardcore, .NET Core is command-line environment friendly.
 
 Visual Studio Code, Visual Studio and Rider will run the launch profile `Bitcoin`.
-This will run a BTCPayServer instance connecting to the services in your Docker service, so you can easily debug and step through the code.
+This will run a **BTCPayServer instance connecting to the services in your Docker service**, so you can easily debug and step through the code.
```

Comment:
```suggestion
This will run a **BTCPay Server instance connecting to the services in your Docker service**, so you can easily debug and step through the code.
```